crypto/keys/sr25519/sr25519_test.go

```diff
@@ -0,0 +1,224 @@
+package sr25519_test
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/crypto"
+	tmsr25519 "github.com/tendermint/tendermint/crypto/sr25519"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sr25519 "github.com/cosmos/cosmos-sdk/crypto/keys/sr25519"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+func TestSignAndValidateSr25519(t *testing.T) {
```

Comment:
In new code we prefer to use testify suites. Could you convert the tests to use a suite?